pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -273,8 +273,7 @@ func syncOne(sj *batchv1beta1.CronJob, js []batchv1.Job, now time.Time, jc jobCo
 	}
 	if tooLate {
 		glog.V(4).Infof("Missed starting window for %s", nameForLog)
-		// TODO: generate an event for a miss.  Use a warning level event because it indicates a
-		// problem with the controller (restart or long queue), and is not expected by user either.
+		recorder.Eventf(sj, v1.EventTypeWarning, "MissedStartingWindow", "Missed starting window for %s", nameForLog)
```

Comment:
* Keep the comment below as a `TODO`. 
* Suggested event reason "MissSchedule"
* No need to print sj's name in event message, it's already part of the event (the first parameter)
* I think it'd be useful to print the missed schedule time too (would it be too long?). Suggested event message: `"Missed scheduled time to start a job: %s", scheduledTime.Format(time.RFC1123Z)`.

Reply:
Done,Thanks for your suggestion